integration-tests/tests/client/challenges.rs

```diff
@@ -613,6 +613,10 @@ fn test_fishermen_challenge() {
 /// If there are two blocks produced at the same height but by different block producers, no
 /// challenge should be generated
 #[test]
+// Something weird happens here. For len_in_blocks = 13 this test passes for versions up to 49,
+// but fails for version 50 (because of chunk validator sampling changes). But if we set it to 20,
+// it fails for version 49 as well
```

Comment:
It is probably worth investigating. While challenges are not enabled by any means, the failure here seems related to your change and not to challenges per se.

Reply:
It fails with len_in_blocks = 20 even without my change

Reply:
I also had problems with that test during working on other issues. It should be investigated

Reply:
@EgorKulikov I suggest creating an issue for this

Reply:
@bowenwang1996 #5914